src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11170,7 +11201,8 @@ private ParameterSyntax ParseLambdaParameter()
             }
 
             SyntaxToken paramName = this.ParseIdentifierToken();
-            var parameter = _syntaxFactory.Parameter(default(SyntaxList<AttributeListSyntax>), modifiers.ToList(), paramType, paramName, null);
+            var exclamation = this.CurrentToken.Kind == SyntaxKind.ExclamationToken ? this.EatToken(SyntaxKind.ExclamationToken) : null;
```

Comment:
can be simplified

Reply:
TryEatToken isn't showing up as available for me. What reference is it under?

Reply:
You will have to merge master into your branch first